### PR TITLE
kb(grid):filter by selected items

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

**Review by yordan-mitev — COMMENTED** [review]
Review suggestions provided in https://github.com/telerik/blazor-docs/pull/1443

File: knowledge-base/grid-filter-by-selected-items.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+---
+title: Filter the Grid by selected items
+description: How to filter the Grid by Selected Items?
+type: how-to
+page_title: Filter the Grid by selected items
+slug: grid-kb-filter-by-selected-items
+position: 
+tags: grid,filter,selected,only
```
Comment (dimodi) [inline knowledge-base/grid-filter-by-selected-items.md:8]: Tags are not keywords, but more like categories. They should make sense in isolation.

```suggestion
tags: grid,filter,selected
```

File: knowledge-base/grid-filter-by-selected-items.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+## Description
+
+How can I achieve the following with the Blazor Grid:
+
+* When the user selects items on different pages, I want to force the Grid to only show the selected items. So I need to filter the Grid by the selected items.
+* If specific items are selected, I want to sort only the selected items.
+
+## Solution
+
+To achieve the desired result, simulate filtering by the selected items:
```
Comment (dimodi) [inline knowledge-base/grid-filter-by-selected-items.md:33]:
```suggestion
To simulate filtering by the selected items:
```

**Review by dimodi — APPROVED** [review]

**Comment by dimodi** [comment]
Consider a See Also section and/or links to the data binding articles in the Common section of the docs.

File: knowledge-base/grid-filter-by-selected-items.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+            }
+            return false;
+        }
+    }
+}
+````
+
+### Data binding through the OnRead event
+
+Bind the Grid [through the OnRead event]({%slug common-features-data-binding-onread%}).
```
Comment (ntacheva) [inline knowledge-base/grid-filter-by-selected-items.md:149]:
```suggestion
Bind the Grid [through the `OnRead` event]({%slug common-features-data-binding-onread%}).
```

**Pull request merged**